account_asset_management/models/account_asset.py

```diff
@@ -316,7 +316,7 @@ def _compute_depreciation(self):
 
     @api.depends("profile_id")
     def _compute_group_ids(self):
-        for asset in self.filtered("profile_id"):
+        for asset in self:
             asset.group_ids = asset.profile_id.group_ids
```

Comment:
```suggestion
        for asset in self:
            if asset.profile_id:
                asset.group_ids = asset.profile_id.group_ids
            else:  # HACK: Not needed in v14 due to odoo/odoo#...
                asset.group_ids = asset.group_ids
```
And check that on v14 is effectively not happening

Reply:
Have you check it in v14 then?

Reply:
In 14.0 is not possible to set asset group in profile but too probably will be necessary to change it according to actual code: https://github.com/OCA/account-financial-tools/blob/14.0/account_asset_management/models/account_asset.py#L314-L317

Reply:
Are you saying that this is a missing feature? Please add such feature then and test the hypothesis of the comment.